tests/expectations/tests/union_dtor.rs

```diff
@@ -52,3 +52,13 @@ fn bindgen_test_layout_UnionWithDtor() {
                 "Alignment of field: " , stringify ! ( UnionWithDtor ) , "::"
                 , stringify ! ( mBar ) ));
 }
+extern "C" {
+    #[link_name = "_ZN13UnionWithDtorD1Ev"]
+    pub fn UnionWithDtor_~UnionWithDtor(this: *mut UnionWithDtor);
```

Comment:
But isn't that a valid identifier? I think Rust accepts all valid Unicode strings....